src/mito2/src/manifest/storage.rs

```diff
@@ -544,6 +619,15 @@ mod tests {
         ManifestObjectStore::new("/", object_store, CompressionType::Uncompressed)
     }
 
+    fn new_checkpoint_metadata_with_version(version: ManifestVersion) -> CheckpointMetadata {
```

Comment:
Looks like it's only used in test.
```suggestion
    #[cfg(test)]
    fn new_checkpoint_metadata_with_version(version: ManifestVersion) -> CheckpointMetadata {
```